report/result.go

```diff
@@ -290,6 +290,7 @@ func ToShell(r probe.Result) string {
 	env["EASEPROBE_ENDPOINT"] = r.Endpoint
 	env["EASEPROBE_STATUS"] = r.Status.String()
 	env["EASEPROBE_TIMESTAMP"] = fmt.Sprintf("%d", r.StartTimestamp)
+	env["EASEPROBE_TIME"] = FormatTime(r.StartTime)
```

Comment:
I'm not quite sure about the environment name as we're not able to easily change it in the future (breaks the backwards comaptibitliy of existing shell scripts) 
Need double check

Reply:
I think the name is fine. 

could you please update the demo shell script and the document?

Reply:
Great. Updated.